ci/build/build-pipeline.yaml

```diff
@@ -514,6 +524,55 @@ spec:
             trigger: true
           - get: tests-image
             passed: [build-proxy-node]
+          - get: eidas-config
+            trigger: true
+
+      - task: generate-config
+        config:
+          platform: linux
+          image_resource: *openjdk_image
+          inputs:
+            - name: eidas-config
+          outputs:
+            - name: eidas-config
+          run:
+            path: /bin/sh
+            args:
+              - -euc
+              - |
+                cd eidas-config/tools
```

Comment:
If the only reason for using `sh` is so that we can change the current working directory, then it may be feasible and less code to use the `dir` option to `run` instead. https://concourse-ci.org/tasks.html#task-run-dir

Reply:
Nifty! I'll update